sdk/storage/storage-file-datalake/test/perfstress/track-2/append.spec.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { StorageDFSTest } from "./storageTest.spec";
+import { DataLakeFileClient } from "../../../src";
+interface StorageDFSAppendTestOptions {
+  size: number;
+}
+
+export class StorageDFSAppendTest extends StorageDFSTest<StorageDFSAppendTestOptions> {
+  public options: PerfStressOptionDictionary<StorageDFSAppendTestOptions> = {
+    size: {
+      required: true,
+      description: "Size to append in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 1024
+    }
+  };
+  static fileName = `newfile${new Date().getTime()}`;
+  fileClient: DataLakeFileClient;
+
+  constructor() {
+    super();
+    this.fileClient = this.directoryClient.getFileClient(StorageDFSAppendTest.fileName);
+  }
+
+  public async globalSetup() {
+    await super.globalSetup();
+    await this.fileClient.upload(Buffer.alloc(0));
```

Comment:
I think each parallel instance should append to its own file, rather than all instances appending to the same file, to avoid potential locking in the service.